terraform/modules/logging/alarms/missing-logs/alarms.tf

```diff
@@ -16,10 +16,9 @@ resource "aws_cloudwatch_metric_alarm" "missing_logs_alarm" {
   }
 
   // For 5 minutes
-  // This appears to cause our alarms to trigger at around the 10-12 minute mark
-  evaluation_periods = "5"
 
-  period = "60"
+  evaluation_periods = "1"
+  period = "300"
```

Comment:
I think this is the line that's failing the `terraformat` linting.

Reply:
Ack! My bad!, completely forgot to run terraform plan...